pages/agent/v3/elastic_ci_aws.md.erb

```diff
@@ -167,17 +169,46 @@ The AWS ECR options are powered by an embedded version of the [ECR plugin](https
 
 ## Elastic Stack CI releases
 
-We recommend running the latest release, which is available at `https://s3.amazonaws.com/buildkite-aws-stack/aws-stack.yml`, or on the [releases page](https://github.com/buildkite/elastic-ci-stack-for-aws/releases).
+We recommend running the latest stable release of the CloudFormation template,
+available at `https://s3.amazonaws.com/buildkite-aws-stack/aws-stack.yml`, or a
+specific release from the [releases page](https://github.com/buildkite/elastic-ci-stack-for-aws/releases).
+
+The latest stable release can be deployed to any of our supported AWS Regions.
+
+We publish the most recent build of the stack to
+`https://s3.amazonaws.com/buildkite-aws-stack/master/aws-stack.yml`, along with
+a version for each commit at
+`https://s3.amazonaws.com/buildkite-aws-stack/master/${COMMIT}.aws-stack.yml`.
+
+<!-- vale off -->
+<!-- alex ignore master -->
+
+A master branch release can also be deployed to any of our supported AWS
+Regions.
+
+<!-- vale on -->
 
-We publish the latest build of the stack to `https://s3.amazonaws.com/buildkite-aws-stack/master/aws-stack.yml`, along with a version for each commit in the form of `https://s3.amazonaws.com/buildkite-aws-stack/master/${COMMIT}.aws-stack.yml`.
+Branches are published to a per-branch URL
+`https://s3.amazonaws.com/buildkite-aws-stack/${BRANCH}/aws-stack.yml`.
 
-Branches are published in the form of `https://s3.amazonaws.com/buildkite-aws-stack/${BRANCH}/aws-stack.yml`.
+Branch releases can only be deployed to `us-east-1`.
```

Comment:
This looks like it could catch people out?

Reply:
It could and even caught me out recently. It’s not something I’d look to change just now so I thought it preferable to document 😄